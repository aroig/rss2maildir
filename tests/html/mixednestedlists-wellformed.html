<ul>
    <li>Fun is on the way
        <ol>
            <li>Honest</li>
            <li>It really is
                <ul>
                    <li>And it's fun</li>
                    <li>We should have at least one silly long line to check that wrapping works like we expect otherwise what's the point</li>
                </ul>
            </li>
        </ol>
        <dl>
            <dt>What is HTML2Text?</dt>
            <dd>
                <p>HTML2Text is a funky bit of python that translates HTML fragments in to plain text in a human readable format. It's primary use was for generating the plain text body of email messages generated from rssfeeds as part of rss2maildir.</p>
                <p>Hopefully one day it'll be good enough that other people will want to use it!</p>
            </dd>
            <dt>Who wrote it?</dt>
            <dd>HTML2Text was written by Brett Parker.</dd>
        </dl>
    </li>
</ul>
