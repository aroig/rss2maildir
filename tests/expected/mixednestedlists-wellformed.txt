 * Fun is on the way
     1. Honest
     2. It really is
        * And it's fun
        * We should have at least one silly long line to check that
          wrapping works like we expect otherwise what's the point

   What is HTML2Text?::

   HTML2Text is a funky bit of python that translates HTML fragments
   in to plain text in a human readable format. It's primary use was
   for generating the plain text body of email messages generated from
   rssfeeds as part of rss2maildir.

   Hopefully one day it'll be good enough that other people will want
   to use it!

   Who wrote it?::
       HTML2Text was written by Brett Parker.
